packages/forms/resources/views/components/tabs.blade.php

```diff
@@ -4,7 +4,7 @@
         tab: null,
 
         init: function () {
-            this.tab = this.getTabs()[0]
+            this.tab = this.getTabs()[{{ $getActiveTab() }} - 1]
```

Comment:
Maybe better to stick to using `@js()`?
```suggestion
            this.tab = this.getTabs()[@js($getActiveTab() - 1)]
```